tox.ini

```diff
@@ -127,5 +124,4 @@ commands =
 [testenv:docs-clean]
 skip_install = true
 deps =
-allowlist_externals = rm
-commands = rm -rf {toxinidir}/docs/stubs/ {toxinidir}/docs/_build
+commands = python -c "import shutil; shutil.rmtree('{toxinidir}/docs/stubs/', ignore_errors=True); shutil.rmtree('{toxinidir}/docs/_build', ignore_errors=True)"
```

Comment:
What's the reason removing files from python interpreter?

Reply:
I made this change because I needed to remove the dist directory in the `.pkg` environment that I added above so I was reviewing how this environment removed directories. It was using `rm` which I thought would not work on Windows. Since we know Python should be available, I thought using `shutil.rmtree` would be an alternative that could work on more platforms.